components/nexus-bootstrap/src/main/java/org/sonatype/nexus/bootstrap/jetty/JettyHolder.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.bootstrap.jetty;
+
+import java.util.Map;
+import java.util.concurrent.CountDownLatch;
+
+import org.eclipse.jetty.server.Server;
+
+/**
+ * Jetty holder.
+ *
+ * @since 3.0
+ */
+public class JettyHolder
```

Comment:
minor, but odd name for something that actually holds a Server object.  ServerHolder would probably be a saner name for this.